# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build bdftopcf-build mkfontdir-build"

PKG_CONFIG_PATH=/x11/share/pkgconfig pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.63; am-1.11)
aclocal; autoconf; automake -a -c
./configure --with-fontrootdir=/x11/share/fonts --prefix=/x11

# build
make

# install
install -d $DESTDIR/x11/bin
install bdftruncate $DESTDIR/x11/bin
install ucs2any $DESTDIR/x11/bin
install -d $DESTDIR/x11/lib/pkgconfig
install -m 644 fontutil.pc $DESTDIR/x11/lib/pkgconfig
install -d $DESTDIR/x11/share/aclocal
install -m 644 fontutil.m4 $DESTDIR/x11/share/aclocal
install -d $DESTDIR/x11/share/fonts/util
install -m 644 map-ISO8859-1 $DESTDIR/x11/share/fonts/util

# build.pc
install -d $DESTDIR/x11/share/pkgconfig
cat > $DESTDIR/x11/share/pkgconfig/font-util-build.pc << EOF
Name: font-util
Version: 1.1.1
Description: X font utilities
Requires: $REQUIRES

devel=\\
/x11/lib/pkgconfig/fontutil.pc \\
/x11/share/aclocal/fontutil.m4

exec=\\
/x11/bin/bdftruncate \\
/x11/bin/ucs2any \\
/x11/share/fonts/util/map-ISO8859-1
EOF

exit
--- font-util-1.1.1.orig/configure.ac
+++ font-util-1.1.1/configure.ac
@@ -30,7 +30,6 @@
 m4_ifndef([XORG_MACROS_VERSION],
 	  [m4_fatal([must install xorg-macros 1.3 or later before running autoconf/autogen])])
 XORG_MACROS_VERSION(1.3)
-AM_CONFIG_HEADER(config.h)
 
 XORG_DEFAULT_OPTIONS
 
